pypln/web/settings.py

```diff
@@ -205,5 +193,28 @@
 
 
 ELASTICSEARCH_CONFIG = {
-    'hosts': ['127.0.0.1'],
+    'hosts': config('ELASTICSEARCH_HOSTS', cast=Csv()),
 }
+
+SERVE_MEDIA = config('SERVE_MEDIA', default=False, cast=bool)
+
+ROUTER_API = config('ROUTER_API')
+ROUTER_BROADCAST = config('ROUTER_BROADCAST')
+ROUTER_TIMEOUT = 5
```

Comment:
We should have removed this a long time ago (pypln.backend doesn't have a router anymore). Maybe this is a good time to finally do it?